website/docs/reference/integrations/jira-cloud-plugin-usage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Jira Cloud Integration - Usage
+---
+
+With the Unleash Jira Cloud Plugin you can create, view and manage Unleash feature toggles directly from a Jira issue.
+
+The plugin also shows you current status of connected toggles.
+
+## Using the plugin
+
+To use the plugin, you'll need to create an issue or use an existing one. Once the issue is saved and you open up the issue panel, you'll be greeted with a button to activate the Unleash plugin for that issue.
+
+![Jira Cloud - Button to activate Unleash plugin in Jira Issue](/img/jira-cloud-activate-unleash-panel-button.png)
+
+### Connecting a toggle to an issue
+
+![Jira Cloud - Button that opens modal to connect toggles to Jira Issue](/img/jira-cloud-issue-button.png)
+
+Clicking this button opens a dialog where you first select an Unleash Project to connect toggle from,
+
+![Jira Cloud - Dropdown to select Unleash project](/img/jira-cloud-select-project-expanded.png)
```

Comment:
Does it contain all projects? or only some?

```suggestion
![Jira Cloud: connect feature toggle form. The project selection dropdown contains all Unleash projects](/img/jira-cloud-select-project-expanded.png)
```

Reply:
It's all projects, nice suggestion